cfgov/jinja2/v1/_includes/molecules/global-search.html

```diff
@@ -17,21 +17,32 @@
         </span>
     </button>
     <div class="m-global-search_content" aria-expanded="false">
-        <form class="m-global-search_content-form">
+        <form class="m-global-search_content-form"
+              action="http://search.consumerfinance.gov/search"
+              method="get">
+            {# Required hidden input for CFPB search. #}
+            <input type="hidden"
+                           name="utf8"
+                           value="✓">
```

Comment:
My guess is that's how it knows it's UTF-8. If that doesn't make it through, it's not.